Date: 2017-12-09 10:52:29,489 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 11:19:46,822 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 11:49:44,147 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 12:09:01,909 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 13:22:00,015 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 13:24:49,007 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 13:27:36,945 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 13:34:48,747 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 13:35:57,600 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 13:36:45,523 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 13:57:29,712 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 14:06:13,160 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 14:36:14,010 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 14:39:40,361 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 14:53:58,617 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 14:57:32,945 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 15:14:49,847 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 15:15:28,971 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 16:11:52,674 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 16:30:32,819 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 16:32:08,962 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 16:34:26,809 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 16:36:29,788 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 16:39:04,248 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 16:45:41,879 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 16:47:23,183 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 16:56:36,282 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 17:44:10,495 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 17:51:54,729 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 17:53:06,194 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 17:53:55,822 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [14] 
Logger: LanguageBuilder.Web.Program 

Exception: System.InvalidOperationException: The instance of entity type 'IdentityUserRole<string>' cannot be tracked because another instance with the same key value for {'UserId', 'RoleId'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(TKey key, InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.NullableKeyIdentityMap`1.Add(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.StartTracking(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry.SetEntityState(EntityState oldState, EntityState newState, Boolean acceptChanges)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry.SetEntityState(EntityState entityState, Boolean acceptChanges, Boolean forceStateWhenUnknownKey)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityGraphAttacher.PaintAction(EntityEntryGraphNode node)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityEntryGraphIterator.TraverseGraph(EntityEntryGraphNode node, Func`2 handleNode)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityGraphAttacher.AttachGraph(InternalEntityEntry rootEntry, EntityState entityState, Boolean forceStateWhenUnknownKey)
   at Microsoft.EntityFrameworkCore.DbContext.SetEntityState(InternalEntityEntry entry, EntityState entityState)
   at Microsoft.EntityFrameworkCore.DbContext.SetEntityState[TEntity](TEntity entity, EntityState entityState)
   at Microsoft.EntityFrameworkCore.DbContext.Add[TEntity](TEntity entity)
   at Microsoft.EntityFrameworkCore.Internal.InternalDbSet`1.Add(TEntity entity)
   at LanguageBuilder.Services.Implementations.RolesService.<UpdateAsync>d__12.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\RolesService.cs:line 127
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Areas.Admin.Controllers.UsersController.<EditRoles>d__7.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Areas\Admin\Controllers\UsersController.cs:line 106
 
Message: The instance of entity type 'IdentityUserRole<string>' cannot be tracked because another instance with the same key value for {'UserId', 'RoleId'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values. 

Query: (null) 



--------------------------------------



Date: 2017-12-09 17:56:30,900 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 17:59:39,423 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 18:03:30,019 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 18:06:02,795 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 18:08:40,537 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 18:13:44,312 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 18:21:17,523 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 18:22:50,573 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 18:24:15,521 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 18:47:01,210 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-09 18:48:22,273 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-10 10:44:20,933 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-10 10:46:46,718 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-10 10:48:29,367 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-10 10:54:53,993 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [2] 
Logger: LanguageBuilder.Web.Program 

Exception:  
Message: Host started 

Query: (null) 



--------------------------------------



Date: 2017-12-10 15:41:13,852 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [9] 
Logger: LanguageBuilder.Web.Program 

Exception: Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: The UPDATE statement conflicted with the FOREIGN KEY constraint "FK_tbl_User_tbl_Subscription_SubscriptionId". The conflict occurred in database "LanguageBuilderDb", table "dbo.tbl_Subscription", column 'Id'.
The statement has been terminated.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.Execute(IRelationalConnection connection, DbCommandMethod executeMethod, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.ExecuteReader(IRelationalConnection connection, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(Tuple`2 parameters)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ExecutionStrategyExtensions.Execute[TState,TResult](IExecutionStrategy strategy, TState state, Func`2 operation)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IReadOnlyList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IReadOnlyList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at LanguageBuilder.Data.LanguageBuilderDbContext.SaveChanges(Boolean acceptAllChangesOnSuccess) in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Data\LanguageBuilderDbContext.cs:line 94
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at LanguageBuilder.Services.Implementations.UsersService.<UpdateAsync>d__8.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\UsersService.cs:line 121
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Controllers.UsersController.<EditProfile>d__5.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\UsersController.cs:line 70
 
Message: An error occurred while updating the entries. See the inner exception for details. 

Query: (null) 



--------------------------------------



Date: 2017-12-10 15:42:42,222 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [14] 
Logger: LanguageBuilder.Web.Program 

Exception: Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: The UPDATE statement conflicted with the FOREIGN KEY constraint "FK_tbl_User_tbl_Subscription_SubscriptionId". The conflict occurred in database "LanguageBuilderDb", table "dbo.tbl_Subscription", column 'Id'.
The statement has been terminated.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.Execute(IRelationalConnection connection, DbCommandMethod executeMethod, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.ExecuteReader(IRelationalConnection connection, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(Tuple`2 parameters)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ExecutionStrategyExtensions.Execute[TState,TResult](IExecutionStrategy strategy, TState state, Func`2 operation)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IReadOnlyList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IReadOnlyList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at LanguageBuilder.Data.LanguageBuilderDbContext.SaveChanges(Boolean acceptAllChangesOnSuccess) in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Data\LanguageBuilderDbContext.cs:line 94
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at LanguageBuilder.Services.Implementations.UsersService.<UpdateAsync>d__8.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\UsersService.cs:line 121
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Controllers.UsersController.<EditProfile>d__5.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\UsersController.cs:line 70
 
Message: An error occurred while updating the entries. See the inner exception for details. 

Query: (null) 



--------------------------------------



Date: 2017-12-10 15:44:19,793 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [28] 
Logger: LanguageBuilder.Web.Program 

Exception: Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: The UPDATE statement conflicted with the FOREIGN KEY constraint "FK_tbl_User_tbl_Subscription_SubscriptionId". The conflict occurred in database "LanguageBuilderDb", table "dbo.tbl_Subscription", column 'Id'.
The statement has been terminated.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.Execute(IRelationalConnection connection, DbCommandMethod executeMethod, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.ExecuteReader(IRelationalConnection connection, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(Tuple`2 parameters)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ExecutionStrategyExtensions.Execute[TState,TResult](IExecutionStrategy strategy, TState state, Func`2 operation)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IReadOnlyList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IReadOnlyList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at LanguageBuilder.Data.LanguageBuilderDbContext.SaveChanges(Boolean acceptAllChangesOnSuccess) in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Data\LanguageBuilderDbContext.cs:line 94
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at LanguageBuilder.Services.Implementations.UsersService.<UpdateAsync>d__8.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\UsersService.cs:line 121
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Controllers.UsersController.<EditProfile>d__5.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\UsersController.cs:line 70
 
Message: An error occurred while updating the entries. See the inner exception for details. 

Query: (null) 



--------------------------------------



Date: 2017-12-10 15:55:36,218 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [25] 
Logger: LanguageBuilder.Web.Program 

Exception: Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: The UPDATE statement conflicted with the FOREIGN KEY constraint "FK_tbl_User_tbl_Subscription_SubscriptionId". The conflict occurred in database "LanguageBuilderDb", table "dbo.tbl_Subscription", column 'Id'.
The statement has been terminated.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.Execute(IRelationalConnection connection, DbCommandMethod executeMethod, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.ExecuteReader(IRelationalConnection connection, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(Tuple`2 parameters)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ExecutionStrategyExtensions.Execute[TState,TResult](IExecutionStrategy strategy, TState state, Func`2 operation)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IReadOnlyList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IReadOnlyList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at LanguageBuilder.Data.LanguageBuilderDbContext.SaveChanges(Boolean acceptAllChangesOnSuccess) in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Data\LanguageBuilderDbContext.cs:line 94
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at LanguageBuilder.Services.Implementations.UsersService.<UpdateAsync>d__8.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\UsersService.cs:line 121
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Controllers.UsersController.<EditProfile>d__5.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\UsersController.cs:line 70
 
Message: An error occurred while updating the entries. See the inner exception for details. 

Query: (null) 



--------------------------------------



Date: 2017-12-18 19:59:09,572 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [22] 
Logger: LanguageBuilder.Web.Program 

Exception: System.ArgumentException: Column out of range
   at OfficeOpenXml.ExcelRange.ValidateRowCol(Int32 Row, Int32 Col)
   at OfficeOpenXml.ExcelRange.get_Item(Int32 Row, Int32 Col)
   at LanguageBuilder.Web.Areas.Admin.Controllers.UploadController.<Words>d__6.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Areas\Admin\Controllers\UploadController.cs:line 89
 
Message: Column out of range 

Query: (null) 



--------------------------------------



Date: 2017-12-18 21:14:53,128 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [39] 
Logger: LanguageBuilder.Web.Program 

Exception: Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: The DELETE statement conflicted with the REFERENCE constraint "FK_tbl_IdentityUserRole_tbl_User_UserId". The conflict occurred in database "DB_A2C13B_languageBuilderDb", table "dbo.tbl_IdentityUserRole", column 'UserId'.
The statement has been terminated.
   at System.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__108_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.<ExecuteAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.<ExecuteAsync>d__32.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.<ExecuteAsync>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.<ExecuteAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.<ExecuteAsync>d__7`2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.<SaveChangesAsync>d__61.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.<SaveChangesAsync>d__59.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.DbContext.<SaveChangesAsync>d__48.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at LanguageBuilder.Services.Implementations.UsersService.<DeleteAsync>d__9.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\UsersService.cs:line 125
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at LanguageBuilder.Web.Areas.Admin.Controllers.UsersController.<DeleteConfirmed>d__8.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Areas\Admin\Controllers\UsersController.cs:line 129
 
Message: An error occurred while updating the entries. See the inner exception for details. 

Query: (null) 



--------------------------------------



